web/e2e-tests/custom-profile.test.ts

```diff
@@ -15,15 +15,15 @@ async function test_add_new_profile_field(page: Page): Promise<void> {
         "Add a new custom profile field",
     );
     assert.strictEqual(
-        await common.get_text_from_selector(page, ".micromodal .dialog_submit_button"),
+        await common.get_text_from_selector(page, ".micromodal.modal--open .dialog_submit_button"),
```

Comment:
These tests changes are needed because the modal for uppy editor is always present in DOM.

Reply:
Hmm, OK. I think that may be OK, but we should probably write a set of helpers like `common.get_text_within_active_modal(".dialog_submit_button")` that abstract this pattern.